cli/scaletest_test.go

```diff
@@ -3,201 +3,53 @@ package cli_test
 import (
 	"bytes"
 	"context"
-	"encoding/json"
-	"os"
 	"path/filepath"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"github.com/coder/coder/cli/clitest"
 	"github.com/coder/coder/coderd/coderdtest"
-	"github.com/coder/coder/codersdk"
 	"github.com/coder/coder/pty/ptytest"
-	"github.com/coder/coder/scaletest/harness"
 	"github.com/coder/coder/testutil"
 )
 
 func TestScaleTestCreateWorkspaces(t *testing.T) {
-	t.Skipf("This test is flakey. See https://github.com/coder/coder/issues/4942")
 	t.Parallel()
 
-	// This test does a create-workspaces scale test with --no-cleanup, checks
```

Comment:
review: already tested in scaletest/createworkspaces/run_test.go. No point duplicating.